HW9/AntonKoluh/guess_game_task1.py

```
@@ -0,0 +1,158 @@
+class RectDisplay():
+    def __init__(self, size_x, size_y, x, y, border = None):
+        self.size_x = size_x
+        self.size_y = size_y
+        self.x = x
+        self.y = y
+        self.border = border
+        self.color = pygame.Color('lightskyblue3')
+
+    def draw(self):
+        self.rect = pygame.Rect((GAME_WIDTH-self.size_x)/2 - self.x, self.y, self.size_x, self.size_y)
+        return pygame.draw.rect(screen, self.color, self.rect) if self.border == None else pygame.draw.rect(screen, self.color, self.rect, self.border)
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)